bin/cli.js

```diff
@@ -154,7 +154,7 @@ async.waterfall([
 
     if (_.includes(documentedVersions, version)) {
       console.log(`Omitting: ${version}`);
```

Comment:
Good catch! Btw do we even need this 'Omitting' message here? if no versions are found `versionist` will exit anyway with the message that describes what went wrong.